docs/containers.conf.5.md

```diff
@@ -219,6 +219,10 @@ Options are:
 Maximum number of processes allowed in a container. 0 indicates that no limit
 is imposed.
 
+**prepare_on_create**=false
+
+Copy up the content from underlying image into the newly created volume when the container is created as opposed to when it is started. If `false`, the container engine will not copy up the content until the container is started. Setting it to `true` may have negative performance implications.
```

Comment:
```suggestion
Copy the content from the underlying image into the newly created volume when the container is created instead of when it is started. If `false`, the container engine will not copy the content until the container is started. Setting it to `true` may have negative performance implications.
```